engine/function/src/templates/Basic.ftl

```diff
@@ -1688,6 +1714,14 @@ public class Basic {
         return ifelse(new BooleanVectorDirect(condition), trueCase, falseCase);
     }
 
+    /**
+     * Returns an array with null values replaced with preceding non-null value
+     *
+     * @param values values.
+     * @return An array of ${pt.primitive} whose values are determined by the preceding elements.
```

Comment:
@return A copy of the specified array, with Deephaven null elements replaced as described above. If the specified array is null, returns null.